src/components/filter/RoomFilter.js

```diff
@@ -0,0 +1,45 @@
+import React  from 'react';
+import FilterItem from './RoomFilterItem';
+import '../../style/TopStyle/Form.css';
+
+
+
+function minusParent(event) {
+    event.target.classList.remove('counter-plus-minus-blue');
+    event.target.classList.add('counter-plus-minus-grey');
+}
+
+
+function RoomFilter() {
```

Comment:
come to one writing components. Use either function declaration either arrow function (I think this way is better)
P.S rewatch other places with function declarations